src/interfaces.ts

```diff
@@ -18,6 +18,7 @@ export interface CosmasOptions {
     defaultLevel?: pino.LevelWithSilent;
     disableStackdriverFormat?: boolean;
     streams?: CosmasStream[];
+    formatters?: object;
```

Comment:
Same goes here for the types

Reply:
Fixed in c772474e33fb04625ca748dbbd747474842e223f